uk.co.johngabriel.amaya.yml

```diff
@@ -0,0 +1,33 @@
+id: uk.co.johngabriel.amaya
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.3'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+
+modules:
+  - name: amaya
+    buildsystem: simple
+
+    sources:
+      - type: git
+        url: https://github.com/cassidyjames/amaya-maiden-of-the-storm.git
```

Comment:
is this a fork? or why not use the canonical git repo? https://bitbucket.org/JohnGabrielUK/amaya-maiden-of-the-storm/src/master/

Reply:
if fork, are they ok with using the same name icon etc. ?

Reply:
I initially didn't know I could make a Bitbucket profile and submit a PR upstream; I've since done that. My GitHub repo is not a fork from the upstream project intended to make a diverging game, but is a git import that has been updated to the latest Godot Engine (and with the Desktop Entry and MetaInfo added, though that could be shipped in the manifest repo if required).

Unfortunately exporting a game with a new Godot Engine version is not just a matter of changing one line in a config file; the engine ports all of the project files forward, making line-by-line changes across the project that are not trivial to just apply with a patch.

If upstream can merge the update to Godot Engine 4.3, then I'd have no problem using the upstream repo directly. But there was a lot of work on the Linux graphics stack for Godot Engine 4.3 and it would be a shame to never be able to take advantage of that (e.g. native Wayland support)—plus the game apparently was originally built in Godot Engine 4.1 which I don't see a BaseApp for.

Reply:
Forking is fine, but then you are publishing it under someone else's rdns and parallel forks should use separate branding etc.

Someone might want to submit the original project at a later point

Reply:
I'll wait to hear from the upstream developer, then; I'd prefer just helping them submit the project and merge this stuff upstream if they're receptive.